src/db/init.sql

```diff
@@ -0,0 +1,49 @@
+CREATE TABLE empleados (
```

Comment:
Issue: You're naming your table in spanish and plural, then you create another table in english and singular (Task). So option option is to name them all in english and plural: employees and tasks. That is a good practice.